common/api/imodeljs-backend.api.md

```diff
@@ -1859,6 +1859,9 @@ export class Entity implements EntityProps {
     forEachProperty(func: PropertyCallback, includeCustom?: boolean): void;
     id: Id64String;
     iModel: IModelDb;
+    static is(otherClass: typeof Entity): boolean;

```

Comment:
did you mean to make this `public`?

Reply:
made it beta for 2.x. Please resolve if it looks good

Reply:
hmmm I don't see the beta tag on this file, let me check

Reply:
Make sure you `npm run build` before `npm run extract-api`.

Reply:
I did it correctly originally, but github's UI didn't show me "outdated" or update my commit range in the "files" tab.
It's there, please review